Agreements/Internal/Solent_TCSW.txt

```diff
@@ -1,11 +1,12 @@
-COPX:EGHI:*:*:NORRY:*:SOLENT:London TC SW:10000:*:^NORRY
+COPX:EGHI:*:*:NORRY:*:SOLENT:London TC SW:7000:*:^NORRY
+COPX:EGHH:*:*:NORRY:*:SOLENT:London TC SW:10000:*:^NORRY
 
-COPX:EGHI:*:TABEN:*:*:SOLENT:London TC SW:10000:*:^TABEN
+COPX:EGHI:*:TABEN:*:*:SOLENT:London TC SW:7000:*:^TABEN
 COPX:EGHH:*:TABEN:*:*:SOLENT:London TC SW:10000:*:^TABEN
 
 COPX:EGHI:*:PEPIS:CPT:*:SOLENT:London TC SW:10000:*:^PEPIS
 
-COPX:EGHI:*:GWC:*:*:SOLENT:London TC SW:10000:*:^GWC
+COPX:EGHI:*:GWC:*:*:SOLENT:London TC SW:8000:*:^GWC
 COPX:EGHH:*:GWC:*:*:SOLENT:London TC SW:10000:*:^GWC
 
 COPX:EGLF:*:*:CPT:*:SOLENT:London TC SW:7000:*:HDG
```

Comment:
Please make the label "coord" to indicate these are not agreements/need coordinating